### PR TITLE
Styling Updates for Admin Reports Pages

### PR DESCRIPTION
**Pull request by vvj5**
-notes in commits

### CODE REVIEW

File: app/views/admin/reports_meetings.html.erb [hunk 1]
```diff
@@ -1,7 +1,14 @@
+<!-- DataTables CSS -->
+<link rel="stylesheet" type="text/css" href="//cdn.datatables.net/plug-ins/1.10.7/integration/bootstrap/3/dataTables.bootstrap.css">
+
+<!-- DataTables -->
+<script type="text/javascript" charset="utf8" src="//cdn.datatables.net/plug-ins/1.10.7/integration/bootstrap/3/dataTables.bootstrap.js"></script>
```
Comment (jah2488) [inline app/views/admin/reports_meetings.html.erb]: We shouldn't have these CDN links in multiple files.

File: app/views/admin/reports_rooms.html.erb [hunk 2]
```diff
@@ -1,36 +1,81 @@
+<!-- DataTables CSS -->
+<link rel="stylesheet" type="text/css" href="//cdn.datatables.net/plug-ins/1.10.7/integration/bootstrap/3/dataTables.bootstrap.css">
+
+<!-- DataTables -->
+<script type="text/javascript" charset="utf8" src="//cdn.datatables.net/plug-ins/1.10.7/integration/bootstrap/3/dataTables.bootstrap.js"></script>
```
Comment (jah2488) [inline app/views/admin/reports_rooms.html.erb]: We shouldn't have these CDN links in multiple files. 

Instead they should be abstracted into a partial or placed inside the `application.html`

File: app/views/admin/reports_rooms.html.erb [hunk 3]
```diff
@@ -1,36 +1,81 @@
...
+
+
+
 <div class="report-inner">
   <h1>TOP ROOMS</h1>
-    <table class="report-container">
-      <table id="top_rooms">
+    <table display>
```
Comment (jah2488) [inline app/views/admin/reports_rooms.html.erb]: What is `<table display>` ?

File: app/views/admin/reports_rooms.html.erb [hunk 4]
```diff
@@ -1,36 +1,81 @@
...
+          </div>
         </thead>
 
         <tbody>
           <% @top_rooms.each do |top_rooms| %>
             <tr>
-            <td><%= top_rooms.first.name %></td>
+            <td>
+                <%= top_rooms.first.name %>
+              </div>
```
Comment (jah2488) [inline app/views/admin/reports_rooms.html.erb]: This `</div>` doesn't have an opening div anywhere and needs to be removed.

File: Gemfile [hunk 5]
```diff
@@ -2,39 +2,60 @@ source "https://rubygems.org"
...
 gem "rails", "~> 4.2"
 gem "pg", "~> 0.18"
 gem "sass-rails", "~> 5.0"
 gem "uglifier", "~> 2.7"
 gem "coffee-rails", "~> 4.1"
+gem "devise", "~> 3.5"
+gem "figaro", "~> 1.1"
 gem "jquery-datatables-rails", github: 'rweng/jquery-datatables-rails'
 gem "jquery-rails", "~> 4.0"
+gem "jquery-ui-rails"
```
Comment (jah2488) [inline Gemfile]: I thought we removed `jquery-ui-rails` ?

File: Gemfile [hunk 6]
```diff
@@ -2,39 +2,59 @@ source "https://rubygems.org"
...
   gem 'database_cleaner'
   gem 'guard' # NOTE: this is necessary in newer versions
   gem 'guard-minitest'
   gem 'guard-cucumber'
+  gem "meta_request", "~> 0.3"
+  gem "pry", "~> 0.10"
+  gem "spring", "~> 1.3"
+  gem "web-console", "~> 2.1"
+end
+
```
Comment (BenHand) [inline Gemfile:16]: duplicate gems listed

File: Gemfile [hunk 7]
```diff
@@ -2,39 +2,59 @@ source "https://rubygems.org"
...
+group :test do
+  gem "byebug", "~> 5.0"
+  gem 'cucumber-rails', :require => false
+  gem 'database_cleaner'
+  gem "faker", "~> 1.4"
+  gem "meta_request", "~> 0.3"
+  gem "pry", "~> 0.10"
+  gem "spring", "~> 1.3"
+  gem "web-console", "~> 2.1"
 end
```
Comment (jah2488) [inline Gemfile]: Lets remove this test group.

File: app/assets/javascripts/application.js [hunk 8]
```diff
@@ -44,12 +44,6 @@ var ready = function() {
   setTimeout(setTime, 0);
 };
 
-$(document).ready(ready);
-$(document).on("page:load", ready);
```
Comment (jah2488) [inline app/assets/javascripts/application.js:48]: oh no? Why gone?

**Pull request merged**